plugins/fabrik_element/fileupload/fileupload.php

```diff
@@ -1066,8 +1062,8 @@ protected function _getAllowedExtension()
 
 		if ($allowedFiles != '')
 		{
-			// $$$ hugh - strip spaces, as folk often do ".foo, .bar"
-			preg_replace('#\s+#', '', $allowedFiles);
+			// $$$ hugh - strip spaces and leading ., as folk often do ".bmp, .jpg"
+			preg_replace('#(\s+|^)\.?#', '', trim($allowedFiles));
```

Comment:
Yes, which I believe just replacing the \s+ with \s* should do.  Test that, and let us know.  Then we can merge this, as I think the media capture stuff all looks good.